config/auth.go

```diff
@@ -65,9 +65,7 @@ func (c *AuthConfig) Merge(o *AuthConfig) *AuthConfig {
 // Finalize ensures there no nil pointers.
 func (c *AuthConfig) Finalize() {
 	if c.Enabled == nil {
-		c.Enabled = Bool(false ||
```

Comment:
Redundant logical OR with false
(This comment applies to multiple changes in this PR)